packages/mobile/src/send/SendAmount.tsx

```diff
@@ -353,14 +353,16 @@ const styles = StyleSheet.create({
   },
   contentContainer: {
     flex: 1,
+    width: Dimensions.get("window").width,
```

Comment:
Is this necessary?

Reply:
i dont think so, i was thinking about how to use Dimensions.get("window") in a way that would work with different screen sizes but wasnt too sure. the 90% width change above will probably suffice!